packages/appstore/src/components/modals/wallet-dummy-success/wallet-dummy-success.scss

```diff
@@ -0,0 +1,71 @@
+.wallet-success-dialog {
+    width: 100%;
+    height: 100%;
+    &__icon,
+    &__title,
+    &__description {
+        text-align: center;
+    }
+
+    &__icon {
+        margin: auto;
+        width: fit-content;
+    }
+
+    &__title {
+        padding-top: 2.5rem;
+    }
+
+    &__description {
+        padding-top: 1.5rem;
+    }
+
+    &__footer {
+        justify-content: center;
+        padding: 0;
+        margin-top: 0;
+        padding-bottom: 1rem;
+
+        @include mobile {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+
+        & .dc-btn {
+            margin: 0.4rem;
+            height: 4rem;
+            border-width: 2px;
+            text-align: center;
+
+            @include mobile {
+                &:first-child,
+                &:last-child {
+                    width: 100%;
+                    margin: 0.4rem 1.4rem 0.4rem 0;
+                }
+            }
+        }
+    }
+}
+
+// Had to resolve to this because wrapper_classname on MobileDialog doesn't work
+.dc-mobile-dialog__content {
```

Comment:
Modifying the .dc-mobile-dialog__content class directly also applies it to anywhere else where we are using the MobileDialog causing unintended behavior, refrain from modifying the existing classes.
you can check this comment where i changed the implementation.
https://github.com/binary-com/deriv-app/pull/10192/commits/b1686376ff13c6db7ba942862deca180d6e5f650

Reply:
Will make the changes accordingly. Thanks for pointing it out @arshad-rao-deriv 🙌🏼